### PR TITLE
[FIX] Build instruction for develop-2.x branch

### LINKED ISSUE #780: [DOCS] Build instruction for develop-2.x are incomplete
The build instruction of develop-2.x branch is incomplete as it doesn't specify how to get core and common jar file in Rocket.Chat.Android/app/libs directory.

I would like to work on this issue.

### PR DESCRIPTION
**Pull request by joeljacobdev**
I have successfully updated the build instruction for develop-2.x branch in Readme.

Closes #780

### CODE REVIEW

**Comment by aniketsingh03** [comment]
@pcforgeek this line in the `build.gradle` already does this task for you. 
https://github.com/RocketChat/Rocket.Chat.Android/blob/befa16fcbb18a5c862222ff0303c2ea2f6cc7f62/app/build.gradle#L129
You do not need to run the script manually. Only importing it in android studio does the job for you!

So, instead of adding the lines to run the script, you can shift the lines that say 'import the project in android studio' above the line which says 'Two `jar` files will be found in this directory (the `common` and `core` jar files), this indicates that the SDK was built correctly.'

File: README.md [hunk 1]
```diff
@@ -19,18 +19,14 @@ Since both the versions use `kotlin` for some or all of their classes, following
...
+    - **Using Script(recommeded)** - Make sure that the android repository and the kotlin sdk have the same immediate parent directory. Change the current directory to `Rocket.Chat.Android/app` and run the `build-sdk.sh` which will result in creating of the required jar file `core*.jar` and `common*.jar` in `Rocket.Chat.Android/app/libs`,by the following steps in your terminal window:
 ```
-cd ..
-cd Rocket.Chat.Android/app/libs
-ls
+cd Rocket.Chat.Android/app
+./build-sdk.sh
 ```
-Two `jar` files will be found in this directory (the `common` and `core` jar files), this indicates that the SDK was built correctly.
-- After the SDK has been built successfully, import the project in Android Studio and build it by following `Build > Make Project`.
```
Comment (aniketsingh03) [inline README.md:33]: Just move this line above line number 32

Comment (joeljacobdev) [reply README.md:33]: If the build.gradle(app) already run the build-sdk.sh then we only need to import the Rocket.Chat.Android to start the project. Are you able to get it working by doing so? 

Please check it on your system by removing both the directories - Rocket.Chat.Android and Rocket.Chat.Kotlin.SDK and again set up your environment by importing only Rocket.Chat.Android. 

You can check if it has successfully imported(even though the jar files are present in libs directory they don't work in my case when I set up my environment by importing the project) by trying to call any API method of SDK inside the project.

Comment (aniketsingh03) [reply README.md:33]: Yeah that seems like an issue, the sdk needs to be built manually in order to work correctly. It works fine after doing a gradle sync though!

**Comment by rafaelks** [comment]
@luciofm Is this change still required?

**Pull request merged**